Address relay traffic analysis (#146)
ietf-wg-ohai/oblivious-http: draft-ietf-ohai-ohttp.md

rate might choose to authenticate the relay to enable the higher rate.

### Traffic Analysis {#ta}

This document assumes that all communication between different Oblivious Client,

> Gateway, and Relay is protected by HTTPS.  This protects information about which

Reviewer: ```suggestion
Oblivious Relay Resource, and Oblivious Gateway Resource is protected by HTTPS.  This protects information about which
```

overall time clients take to receive a response, which might not be what some
clients want.

A relay that forwards large volumes of exchanges can provide better privacy by
providing larger sets of messages that need to be matched.

> Traffic analysis is not restricted to network observers. A malicious relay could

Reviewer: ```suggestion
Traffic analysis is not restricted to network observers. A malicious Oblivious Relay Resource could
```

use traffic analysis to learn information about otherwise encrypted requests

> and responses relayed between clients and gateways. Since relays terminate

Reviewer: ```suggestion
and responses relayed between clients and gateways. An Oblivious Relay Resource terminates
```

> TLS connections from clients, they see message boundaries. This privileged

Reviewer: ```suggestion
TLS connections from clients, so they see message boundaries. This privileged
```
